### PR TITLE
#157752286: approve a request by an admin 

### PR DESCRIPTION
**Pull request by TMDav007**
#### What does this PR do?
- This PR approves a request by an admin

#### Description of Tasks to be completed?
-  approves a request by an Admin
- Athenticate an admin
- Add endpoint to route
- Test the route

#### How should this be manually tested?
- clone this your local machine https://github.com/TMDav007/Maintenance-Tracker.git
- navigate to the directory
- run $ npm install
- Run app locally using $ npm run start
- Using postman
- Test for the PUT with this  URL http://localhost:8000/api/v1/requests/

- Run ```npm run test``` to test with mocha

#### What are the relevant pivotal tracker stories?
    - story type: feature
    - story id: 157752286
    - story title: admin approve a request

#### Any background context you want to provide?
- None

#### What are the relevant Github Issues?
- None

#### Questions:
- None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "babel-node server/server.js",
-    "start:dev": "nodemon --watch server --exec babel-node server/server.js",
+    "start:dev": "set NODE_ENV=development&& nodemon --watch server --exec babel-node server/server.js",
```
Comment (paradisekelechi) [inline package.json:8]: Does this work for you very well?
If not, use cross-env

File: server/controller/AdminController.js [hunk 2]
```diff
@@ -49,6 +49,64 @@ class AdminController {
       });
     }
   }
+
+  /**
+ *@desc approve a requests by an admin
+ *
+ * @param {string} req
+ * @param {string} res
```
Comment (paradisekelechi) [inline server/controller/AdminController.js]: So req is not a string. For you to use a dot operator on it, what do you think it is?

File: server/controller/RequestsController.js [hunk 3]
```diff
@@ -205,6 +205,102 @@ class RequestsController {
       });
     }
   }
+
+  /**
+ *@desc  it create creates a requests
+ *
+ * @param {string} req
```
Comment (paradisekelechi) [inline server/controller/RequestsController.js]: Not a string

File: server/route/routes.js [hunk 4]
```diff
@@ -40,6 +41,11 @@ const routes = (app) => {
 
   // admin
   app.get('/api/v1/requests', authenicateAdmin, getAllRequests);
+  app.put('/api/v1/users/requests/:id', updateARequest);
+
+  // admin
```
Comment (paradisekelechi) [inline server/route/routes.js]: Wrong comment structure

File: package.json (hunk 1 above)
Comment (TMDav007) [reply package.json:8]: yes, it worked fine

**Pull request merged**